ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/fonts/FontResourceProvider.kt

```diff
@@ -0,0 +1,19 @@
+package com.revenuecat.purchases.ui.revenuecatui.fonts
+
+import androidx.annotation.FontRes
+import com.revenuecat.purchases.ui.revenuecatui.activity.PaywallActivityLauncher
+
+/**
+ * Implement this interface to provide custom fonts to the [PaywallActivityLauncher].
+ * If you don't, the default material3 theme fonts will be used.
+ */
+interface FontResourceProvider {
```

Comment:
Since `FontFamily` is not parcelizable/serializable, we can't pass it to the PaywallActivity. Instead, I'm adding this that will allow to set the font as long as it's bundled. It will not work for downloaded fonts though, but I was thinking we can add support for that later if needed.